app/src/main/java/com/android/shelfLife/ui/overview/AddFoodItem.kt

```diff
@@ -0,0 +1,217 @@
+package com.android.shelfLife.ui.overview
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material3.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.android.shelfLife.model.foodFacts.FoodCategory
+import com.android.shelfLife.model.foodFacts.FoodFacts
+import com.android.shelfLife.model.foodFacts.FoodUnit
+import com.android.shelfLife.model.foodFacts.Quantity
+import com.android.shelfLife.model.foodItem.FoodItem
+import com.android.shelfLife.model.foodItem.FoodStatus
+import com.android.shelfLife.model.foodItem.FoodStorageLocation
+import com.android.shelfLife.model.foodItem.ListFoodItemsViewModel
+import com.android.shelfLife.model.household.HouseholdViewModel
+import com.android.shelfLife.ui.navigation.NavigationActions
+import com.google.firebase.Timestamp
+import java.text.SimpleDateFormat
+import java.util.*
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun AddFoodItemScreen(
+    navigationActions: NavigationActions,
+    houseHoldViewModel: HouseholdViewModel,
+    foodItemViewModel: ListFoodItemsViewModel
+) {
+
+  var foodName by remember { mutableStateOf("") }
+  var amount by remember { mutableStateOf("") }
+  var unit by remember { mutableStateOf(FoodUnit.GRAM) }
+  var category by remember { mutableStateOf(FoodCategory.OTHER) }
+  var location by remember { mutableStateOf(FoodStorageLocation.PANTRY) }
+  var expireDate by remember { mutableStateOf("") }
+  var openDate by remember { mutableStateOf("") }
+  var buyDate by remember { mutableStateOf(formatTimestampToDate(Timestamp.now())) }
+
+  var unitExpanded by remember { mutableStateOf(false) }
+  var categoryExpanded by remember { mutableStateOf(false) }
+  var locationExpanded by remember { mutableStateOf(false) }
+
+  Scaffold(
+      modifier = Modifier.fillMaxSize(),
+      topBar = { TopAppBar(title = { Text("Add Food Item") }) },
```

Comment:
It's a good habit to develop to avoid writing hard coded string in the code, and favor the use of String.xml resources. This will makes it easier to later [localize](https://developer.android.com/guide/topics/resources/localization) the application. It also makes the app more maintainable as it is easy to modify one text, without having to remember to modify every places that are displaying the hard coded text.

This applies to every other `Text` in other `composables`.